contractcourt/channel_arbitrator.go

```diff
@@ -859,6 +865,25 @@ func (c *ChannelArbitrator) stateStep(
 		case chainTrigger, userTrigger:
 			log.Infof("ChannelArbitrator(%v): noop trigger %v",
 				c.cfg.ChanPoint, trigger)
+
+			// The commitment transaction has been broadcast, but it
+			// doesn't necessarily need to be the commitment
+			// transaction version that is going to be confirmed. To
+			// be sure that we any of those versions can be anchored
+			// down, we now submit all anchor resolutions to the
+			// sweeper. The sweeper will keep trying to sweep all of
+			// them.
+			anchors, err := c.cfg.Channel.NewAnchorResolutions()
+			if err != nil {
+				return StateError, closeTx, err
+			}
+			if anchors != nil {
+				err = c.sweepAnchors(anchors, triggerHeight)
```

Comment:
Perhaps we should make this configurable? As we still attempt to set our commitment fee for a timely confirmation, this'll most likely just end up costing the user additional fees.

Reply:
We try to sweep at the min fee rate here. Assuming 1 sat/b, the sweep will be net positive.